src/App.vue

```diff
@@ -496,6 +497,9 @@ export default {
       // now that we have business info, load state filing
       await this.loadStateFiling()
 
+      // now that we have business info, load the registration filing
+      await this.loadRegistrationFiling()
```

Comment:
Can you make specific to your needs please? -> only call this if you're going to use it for DCB (sole prop, not staff, etc)

Reply:
We'd need this in order to display the EntityMenu, is there a better place this logic can reside?

Reply:
Or are you saying we should just wrap this in a conditional?

Reply:
Yes

Reply:
In fact, I think you could make `loadRegistrationFiling` a store getter and not have to save the restoration filing, etc.

But I'm still hoping we don't need any of this and the user/comp party/prioprietor can be checked in the Legal API and we can use allowable actions to determine whether to display the menu item or not.

Reply:
> But I'm still hoping we don't need any of this and the user/comp party/prioprietor can be checked in the Legal API and we can use allowable actions to determine whether to display the menu item or not.

@argush3 Does this make sense? Does it definitely belong in the Legal API not UI? It is easy to do?

Reply:
> In fact, I think you could make `loadRegistrationFiling` a store getter and not have to save the restoration filing, etc.
> 
> But I'm still hoping we don't need any of this and the user/comp party/prioprietor can be checked in the Legal API and we can use allowable actions to determine whether to display the menu item or not.

My only concern is that the data will be fetched every time is `isAllowed` method is run. In this case it's cached. Unless Pinia does this already

Reply:
Yes, getters are cached.

I'm going to hold off further comments until we see about using allowable actions instead.